src/services/supabase.ts

```diff
@@ -312,15 +260,19 @@ export const supabaseRetry = <T>(makeCall: Function, action: string) => {
             const response = await makeCall();
 
             const error = response.error;
+            const {
+                data: { session },
+                error: authError,
+            } = await supabaseClient.auth.getSession();
```

Comment:
We now need to fetch user from the client with an awaited function as they removed direct access to session.